miv/io/data.py

```diff
@@ -185,6 +187,145 @@ def set_channel_mask(self, channel_id: Iterable[int]):
         """
         self.masking_channel_set.update(channel_id)
 
+    def clear_channel_mask(self):
+        """
+        Clears all present channel masks.
+        """
+
+        self.masking_channel_set = set()
+
+    def _auto_channel_mask(
+        self,
+        spontaneous_binned: Tuple[np.ndarray, int, List[int]],
```

Comment:
@jihugo I really wouldn't recommend passing around the tuples of objects; it is confusing. I would recommend splitting them in three, and name each one.

Reply:
6fdf2ea